.github/workflows/ci-test.yml

```diff
@@ -125,6 +144,314 @@ jobs:
           exit $success
         env:
           DOCKER_CONTENT_TRUST_REPOSITORY_PASSPHRASE: ${{ secrets.DOCKERHUB_KEY_PASSWORD }}
+      - name: push to novi ecr
+        run: |
+          #push to novi ecr with standard names
+          BRANCH="$CHANGES_TARGET_BRANCH"
+          GIT_REV=$(git rev-parse --short=8 HEAD)
+          aws ecr get-login-password --region ${{ secrets.AWS_REGION }} | \
+          docker login --username AWS --password-stdin "${AWS_ECR_ACCOUNT_URL}"
+          docker/docker_republish.sh -t pre_${BRANCH}_${GIT_REV} -o land_${GIT_REV} -r ${AWS_ECR_ACCOUNT_URL} -d
+
+  land-blocking-test:
+    name: Run land blocking test
+    runs-on: self-hosted
+    needs: build-images
+    # NOTE the total time should cover build (~10 min) and test (~20 min).
+    # The additional time can cover the retries and wait.
+    timeout-minutes: 60
+    steps:
+      - uses: actions/checkout@v1
+      - name: Get PR number
+        id: pr-num
+        uses: actions/github-script@v3
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          result-encoding: string
+          script: |
+            // Find the number of the pull request that trigggers this push
+            let pr_num = 0;
+            let commit_message = context.payload.head_commit.message;
+            let re = /.*[^]Closes:\s\#(\d+)$/;
+            if (re.test(commit_message)) {
+              let match = re.exec(commit_message);
+              pr_num = match[1];
+              return pr_num
+            } else {
+              console.log("GH event payload\n", context.payload);
+              throw "Did not find pull request num in commit message. -\\_(O_o)_/-"
+            }
+      - name: Save PR number
+        run: |
+          echo "PR_NUM=${{steps.pr-num.outputs.result}}" >> $GITHUB_ENV
```

Comment:
prepare already has to get this for the auto branch, pretty sure it's already in "changes-pull-request-number"

Reply:
can probably delete the Get PR number and   Save PR number